### DISCUSSION TITLE
Cant login to IP

### DISCUSSION THREAD

**Discussion by 7o7yee** [raspiblitz/raspiblitz#4207] [Q&A] [answered]
Upon logging in after a hiatus, I was prompted to upgrade to 1.9 and since then I have not been able to login through IP, I dont know of any other way to login and Im not seeing any other issues about this anywhere. When I type the IP, I get the Raspiblitz login screen, but it never logs in. If I just wait it out I get 504 timeout error. I am fully synced according to the lil screen on the pi (I have the same exact setup as BTC Sessions youtube tutorial video). I also just out of curiosity, at the login screen just hit enter without a password and it still timesout. I also tried without correct password just to see if it would even tell me incorrect password and it only times out. Ive tried on different computers, I dont know what to do. Also, how do i turn it off now? Is it ok to just hard turn it off with the switch?

**Comment by rootzoll** [comment, marked as answer]
Have you tried to login thru terminal: https://github.com/raspiblitz/raspiblitz#boot-your-raspiblitz

If you are able to login thru terminal: Go UPDATE > RELEASE and then update to v1.10.0 (latest version) and see if you still have those problems with the web login.

**Reply by 7o7yee** [reply]
Thanks for the reply, after reading about this I was able to login in through terminal and shutdown, now I'll flash the new update and try again, thank you, I appreciate it!

After I update I'll confirm my status.

**Reply by 7o7yee** [reply]
Everything is good! Thanks! Had no idea about this ssh method lol
